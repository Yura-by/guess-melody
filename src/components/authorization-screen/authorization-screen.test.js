import AuthorizationScreen from './authorization-screen.jsx';
import React from 'react';
import renderer from 'react-test-renderer';

it(`AuthorizationScreen snapshot test`, () => {
  const tree = renderer
    .create(<AuthorizationScreen
      isBadLoginData={false}
      email={``}
      password={``}
      formSubmitHandler={() => {}}
      loginChangeHandler={() => {}}
      passwordChangeHandler={() => {}}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
