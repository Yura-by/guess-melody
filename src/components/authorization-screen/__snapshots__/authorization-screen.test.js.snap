// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorizationScreen snapshot test 1`] = `
<section
  className="login"
>
  <div
    className="login__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="login__title"
  >
    Необходима авторизация
  </h2>
  <p
    className="login__text"
  >
    Представьтесь!
  </p>
  <form
    action=""
    className="login__form"
    onSubmit={[Function]}
  >
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="name"
      >
        Логин
      </label>
      <input
        className="login__input"
        id="name"
        name="name"
        onChange={[Function]}
        type="text"
        value=""
      />
    </p>
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="password"
      >
        Пароль
      </label>
      <input
        className="login__input"
        id="password"
        name="password"
        onChange={[Function]}
        type="text"
        value=""
      />
      <span
        className="login__error"
        style={Object {}}
      >
        Неверный пароль или неверная почта
      </span>
    </p>
    <button
      className="login__button button"
      type="submit"
    >
      Войти
    </button>
  </form>
</section>
`;
